concent_api/core/payments/backends/sci_backend.py

```diff
@@ -1,6 +1,8 @@
 from enum import Enum
 
 from golem_sci.blockshelper import BlocksHelper
+from web3 import Web3
+
```

Comment:
If we're doing this, we should add web3 to our dependencies. And remove it later when we stop using it.

Reply:
Added to `requirements.txt`